templates/addons/env/opensearch.yml

```diff
@@ -1,11 +1,24 @@
 Parameters:
+  # Copilot required Parameters...
   App:
     Type: String
     Description: Your application's name.
   Env:
     Type: String
     Description: The environment name your service, job, or workflow is being deployed to.
+
```

Comment:
I don't think this is necessary.  Does Opensearch have any sequencing issues?

Reply:
I wouldn't know, I haven't had to deploy/delete one yet, but I expect it would trip up on `EnvironmentSecurityGroup` like the Aurora Postgres addon did and, to me anyway, it makes sense refer to things from the environment stack in a consistent way in all the addons.